File: systems-for-design-and-implementation/labs/lab3/LibraryNetworking/src/library/network/dto/BorrowDTO.scala
package library.network.dto

import java.io.Serializable

import library.model.{Book, User}

class BorrowDTO(var book: Book, var user: User) extends Serializable
{
    def getBook: Book =
    {
        book
    }

    def setBook(book: Book)
    {
        this.book = book
    }

    def getUser: User =
    {
        user
    }

    def setUser(user: User)
    {
        this.user = user
    }

    override def toString: String =
    {
        "BorrowDTO{" + "book=" + book + ", user=" + user + '}'
    }
}

File: systems-for-design-and-implementation/labs/lab2/LibraryNetworking/src/library/network/dto/BorrowDTO.java
package library.network.dto;

import library.model.Book;
import library.model.User;

import java.io.Serializable;

public class BorrowDTO implements Serializable
{
    private Book book;
    private User user;

    public BorrowDTO(Book book, User user)
    {
        this.book = book;
        this.user = user;
    }

    public Book getBook()
    {
        return book;
    }

    public void setBook(Book book)
    {
        this.book = book;
    }

    public User getUser()
    {
        return user;
    }

    public void setUser(User user)
    {
        this.user = user;
    }

    @Override
    public String toString()
    {
        return "BorrowDTO{" +
            "book=" + book +
            ", user=" + user +
            '}';
    }
}
